src/Azure/README.md

```diff
@@ -15,6 +15,10 @@ Please see the [Base Installation Guide](https://socialiteproviders.com/usage/),
   'client_id' => env('AZURE_CLIENT_ID'),  
   'client_secret' => env('AZURE_CLIENT_SECRET'),  
   'redirect' => env('AZURE_REDIRECT_URI') 
+  'tenant' => env('AZURE_TENANT_ID'),
+  'scopes' => ['User.Read'], // it can change
```

Comment:
If a specific scope is always required, add it to `protected $scopes` please

Reply:
yes scope is required. `protected $scopes` added